runtime/calamari/src/lib.rs

```diff
@@ -159,7 +160,10 @@ parameter_types! {
 
 impl pallet_tx_pause::Config for Runtime {
 	type Event = Event;
-	type UpdateOrigin = EnsureRoot<AccountId>;
+	type UpdateOrigin = EnsureOneOf<
```

Comment:
I'd propose the following:
-       type UpdateOrigin = EnsureRoot<AccountId>;
+       type UpdateOrigin = EnsureOneOf<
+               EnsureRoot<AccountId>,
+               pallet_collective::EnsureMembers<_2, AccountId, TechnicalCollective>,
+       >;